examples/api/src/stores/loadingState.store.js

```diff
@@ -0,0 +1,10 @@
+import { useState } from "react";
+import { createStore } from "reusable";
+
+export const useLoadingState = createStore(() => {
+  const [loading, setLoading] = useState(false);
```

Comment:
It's common to use 'isLoading' for boolean values (and setIsLoading for the setter function).
There's no need to rename setLoading, you can just use the same name, for example:
```
  const [isLoading, setIsLoading] = useState(false);

  return {
    isLoading,
    setIsLoading
  };
```
```